torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -441,6 +441,34 @@ RegisterOperators reg(
            stack->emplace_back(result);
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::AutogradAllZero(...) -> bool",
```

Comment:
Do we ever expect to call this with a TensorList ? Could we get rid of that branch ? (and maybe just have it take in a `Tensor[]`

Reply:
I was rushing with implementation of it, and I think I might've gotten it wrong. Let me recheck this.